proxy/src/console/messages.rs

```diff
@@ -284,6 +284,7 @@ pub(crate) struct DatabaseInfo {
     /// be inconvenient for debug with local PG instance.
     pub(crate) password: Option<Box<str>>,
     pub(crate) aux: MetricsAuxInfo,
+    pub(crate) allowed_ips: Option<Vec<IpPattern>>,
```

Comment:
nit: Maybe we could use serde(default)? an empty set of IPs is considered as disabled anyway.

Reply:
Done.